### ISSUE TITLE
[bug]: Error on TextChangedI: Request was cancelled

### ISSUE THREAD

**Issue by ProgrammingLife** [neoclide/coc.nvim#2950]
## Result from CocInfo

```
vim version: NVIM v0.5.0-dev+1122-g8f4b9b8b7
node version: v13.14.0
coc.nvim version: 0.0.80-2e550113c3
coc.nvim directory: /home/denis/.vim/plugged/coc.nvim
term: rxvt-unicode-256color
platform: linux
```
With stable release of neovim 0.4.4 I got the same problem.
Tested with VIM 8.2 - the same problem.

## Reproduce the bug

I work with flutter. When I type opening bracket: "(" after the word "Text" in insert mode here:

```
[
	Text(
];
```
I get this error: 

```
[coc.nvim]: Error on TextChangedI: Request was cancelled
Error: Request was cancelled
    at _y (/home/denis/.vim/plugged/coc.nvim/build/index.js:27:224)
    at In (/home/denis/.vim/plugged/coc.nvim/build/index.js:26:12534)
    at Immediate.<anonymous> (/home/denis/.vim/plugged/coc.nvim/build/index.js:26:12384)
    at processImmediate (internal/timers.js:456:21) 
Press ENTER or type command to continue
```

It happens in long code files only. My "widget.dart" has 900 lines of code. With simple ".dart" files (~20 lines) I don't get any errors in the same case.

I'm running file with: `nvim -u ~/.vim/mini.vim lib/screens/login/widget.dart`
```
$ cat ~/.vim/mini.vim
call plug#begin('~/.vim/plugged')
	Plug 'neoclide/coc.nvim', {'branch': 'release', 'for': [ 'typescript', 'javascript', 'html', 'cs', 'dart', 'gd', 'godot', 'yaml', 'python' ]}
call plug#end()

$
```
In `:CocInfo` I have:
```
2021-03-07T04:09:55.177 INFO (pid:6143) [attach] - receive notification: highlight []
2021-03-07T04:11:34.077 INFO (pid:6143) [attach] - receive notification: highlight []
2021-03-07T04:15:06.492 INFO (pid:6143) [attach] - receive notification: highlight []
2021-03-07T04:15:11.597 INFO (pid:6143) [completion-complete] - Results from: around,buffer
2021-03-07T04:15:15.892 ERROR (pid:6143) [events] - Handler Error on TextChangedI Error: Request was cancelled
    at Cy (/home/denis/.vim/plugged/coc.nvim/build/index.js:27:224)
    at In (/home/denis/.vim/plugged/coc.nvim/build/index.js:26:12525)
    at Immediate.<anonymous> (/home/denis/.vim/plugged/coc.nvim/build/index.js:26:12376)
    at processImmediate (internal/timers.js:456:21)
```

neovim & CoC plugins are up to date.

**Comment by Kavantix** [comment]
I’ve seen this happen as well, started when I upgraded coc to 0.80

**Comment by chemzqm** [comment]
Fixed by 02a6977
